sbnobj/Common/SBNEventWeight/EventWeightParameterSet.h

```diff
@@ -21,10 +21,12 @@ struct EventWeightParameter {
 
   /** Constructor specifying all parameter properties. */
   EventWeightParameter(std::string name, float mean, float width, size_t covIndex=0)
-      : fName(name), fMean(mean), fWidth(width), fCovIndex(covIndex) {}
+      : fName(name), fMean(mean), fWidth(width), fCovIndex(covIndex) {
+    fWidths.clear();
```

Comment:
This line is not necessary though. fWidths will automatically be empty when the object is constructed.

Reply:
Ah, just removed this line to make it neater :)